TabloidCLI/UserInterfaceManagers/PostManager.cs

```diff
@@ -0,0 +1,156 @@
+                    return this;
+                case "5":
+                    //NoteManagement();
+                    Console.WriteLine("NoteManagement not implemented yet.");
+                    return this;
+                case "0":
+                    return _parentUI;
+                default:
+                    Console.WriteLine("Invalid Selection");
+                    return this;
+            }
+        }
+
+        private void List()
+        {
+            List<Post> posts = _postRepository.GetAll();
+            foreach (Post post in posts)
+            {
+                Console.WriteLine($"{post.Title} {post.Url}");
+            }
+        }
+
+        private Post Choose(string prompt = null)
+        {
+            if (prompt == null)
+            {
+                prompt = "Please choose a Post:";
+            }
+
+            Console.WriteLine(prompt);
+
+            List<Post> posts = _postRepository.GetAll();
+
+            for (int i = 0; i < posts.Count; i++)
+            {
+                Post post = posts[i];
+                Console.WriteLine($" {i + 1}) {post.Title}");
+            }
+            Console.Write("> ");
+
+            string input = Console.ReadLine();
+            try
+            {
+                int choice = int.Parse(input);
+                return posts[choice - 1];
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Invalid Selection");
+                return null;
+            }
+        }
+
+        private void Add()
+        {
+            Console.WriteLine("New Post");
+            Post post = new Post();
+
+            Console.Write("Title: ");
+            post.Title = Console.ReadLine();
+
+            Console.Write("Url: ");
+            post.Url = Console.ReadLine();
+
+            //Console.Write("Publish DateTime: ");
+            //post.PublishDateTime = Console.ReadLine();
+
+            _postRepository.Insert(post);
+        }
+
+        private void Edit()
+        {
+            Post postToEdit = Choose("Which post would you like to edit?");
+            if (postToEdit == null)
+            {
+                return;
+            }
+
+            Console.WriteLine();
+            Console.Write("New Post Title (blank to leave unchanged: ");
+            string title = Console.ReadLine();
+            if (!string.IsNullOrWhiteSpace(title))
+            {
+                postToEdit.Title = title;
+            }
+            Console.Write("New Post Url (blank to leave unchanged: ");
+            string url = Console.ReadLine();
+            if (!string.IsNullOrWhiteSpace(url))
+            {
+                postToEdit.Url = url;
+            }
+            Console.Write("New Publish DateTime (blank to leave unchanged: ");
+            string dateTime = Console.ReadLine();
+            if (!string.IsNullOrWhiteSpace(dateTime))
+            {
+                //postToEdit.PublishDateTime = dateTime;
```

Comment:
is this supposed to be commented out or active code?

Reply:
In the future tickets I will take care of it. It is just a placeholder for now.

Reply:
perfect! just checking before i rubber stamp it thanks!